.docs/concepts.md

```diff
@@ -0,0 +1,96 @@
+# Application Insights DotNet SDK Concepts
+This lists the high level concepts of our SDK and links to detailed guides to help you get started.
+
+
+To use the Application Insights SDK you must configure an Instrumentation Key which can be [obtained from an Application Insights resource](https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource).
+Or you can use a [Connection string](https://docs.microsoft.com/azure/azure-monitor/app/sdk-connection-string?tabs=net) to simplify the configuration of our endpoints.
+Both an Instrumentation Key and Connection String are provided for you on the Overivew Dashboard of your Application Insights resource.
+
+
+
+## TelemetryClient
+The `TelemetryClient` object is the primary root object for the library. 
+Almost all functionality around telemetry sending is located on this object. 
+
+### Initialization
+You must initialize an instance of this object and populate it with your Instrumentation Key to identify your data.
+
+```C#
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+
+var configuration = new TelemetryConfiguration
+{
+    // Please use either an Instrumentation Key or a Connection String but not both.
```

Comment:
Given the direction is to use Connection String, I think we should remove Instrumentation Key from the conceptual document.

Reply:
I'm trying to ignore my bias here because I implemented Connection Strings. :)

The UX still displays the Instrumentation Key on the Overview blade. Since it's a very simple way to get started with our SDK I think it's still a valid example for getting started. 
I don't feel strongly about it, but most of our official docs still use the ikey example for getting started. 

Maybe this should be a larger refactoring effort for the docs team to review all public docs and official repos?

Reply:
We can start here. Need help from @ank3it to drive other docs to catch up.

The general principle is that we should only give *one recommended path* in `conceptual documents`. It is okay to list _all possible ways_ in `reference document`.

Reply:
Thanks for clarifying conceptual vs reference. I'll keep that in mind going forward!